src/components/RegMenu.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import {
+  getAllRegisters,
+  getCalleeSaved,
+  getCallerSaved,
+} from './helpers/RegMenuHelpers';
+
+const RegMenu = ({ registers }) => {
+  const [filter, setFilter] = useState('all');
+
+  const CurrentCategory = () => (
+    (
+      <>
```

Comment:
this kind of looks like a button group. maybe we could wrap in a div instead of a fragment.

see https://getbootstrap.com/docs/4.4/components/button-group/ for an example of what i mean

Reply:
added this